.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: python
+python:
+- '3.6'
+- '3.5'
+- '3.4'
+
+matrix:
+  include:
+    - python: 3.6
+      sudo: required
+      dist: trusty
```

Comment:
Trusty is a version of Ubuntu. Currently Travis offers "very old Ubuntu" by default and "old Ubuntu" by choosing trusty here. I wen't with trusty as I'm going to do further hacks and was as new ubuntu as possible.